raspbian/applications/camera.md

```diff
@@ -601,6 +712,41 @@ On each press of the Enter key, the recording will be paused or restarted. Press
 
 Sending a `USR1` signal to the `raspivid` process will toggle between recording and paused. This can be done using the `kill` command, as below. You can find the `raspivid` process ID using `pgrep raspivid`.
 
+```
+--split,    -sp
+```
+When in a signal or keypress mode, each time record is restarted, create a new file.
+
+```
+--circular,    -c
+```
+
+Select circular buffer mode. All encoded data is stored in a circular buffer until a trigger is activated, then the buffer is saved. 
+
+```
+--vectors,    -x
+```
+
+Turns on output of motion vectors from the H264 encoder, to the specified filename.
+
+```
+--flush,    -fl
+```
+
+Forces a flush of output data buffers as soon as video data is written. This bypasses any OS caching of written, and can decrease latency.
```

Comment:
`caching of written` -> `caching of written data` ?

Reply:
Sounds right, but will let @JamesH65 put that in, since I don't actually know.